samples/getting-started/go.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
we could also move a more abstract version of this build script to the toolchain or at least uikit

Reply:
yeah I don't see any reason not to put this into the toolchain too!